src/actions/index.js

```diff
@@ -0,0 +1,49 @@
+export const addAllProjects = (projects) => ({
```

Comment:
Maybe change to set or getAllProject/Palettes because we should be only using this once.